ui/css/perf.css

```diff
@@ -37,6 +37,32 @@ h1 {
   }
 }
 
+.btn-info {
+  background-color: #0d7d8f !important;
```

Comment:
I think it'd be confusing to anyone working on this to use the bootstrap/reactstrap "info" class but then in this style sheet overriding that color. If we're not going to use the default info color prop, can we pass in a class name to the button component instead of using a color prop? 

Also a suggestion for whenever you're overriding framework styles is to have a comment to explain why, so future contributors/developers will be able to know why the decision was made.

Reply:
Sure. In order to do that, I would have to change all the `<Button>` which have the `color="info"` prop (that would be around 14 in all apps). And I think it should also have a guideline in the documentation.

Is that ok?

Reply:
yes, that's fine.